Makefile

```diff
@@ -50,15 +50,32 @@ else
 	stack install --pedantic --test --bench --no-run-benchmarks --local-bin-path=dist
 endif
 
-.PHONY: delete-cache-on-linker-errors
-delete-cache-on-linker-errors:
+# Clean
+.PHONY: clean
+clean: delete-cache-on-linker-errors
```

Comment:
So, `clean` depends on `delete-cache-on-linker-errors` and the other way round. Does this work? Looks like an endless loop... :thinking:

Reply:
ah, no, oversight, i was moving things around too violently in between and lost perspective.  will fix it.

Reply:
7b8d18634